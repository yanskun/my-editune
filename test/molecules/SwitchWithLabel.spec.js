import { createLocalVue, shallowMount } from "@vue/test-utils"
import ElementUI from "element-ui"
import Component from "~/components/molecules/SwitchWithLabel"

const localVue = createLocalVue()
localVue.use(ElementUI)

describe("SwitchWithLabel", () => {
  test("snapshot", () => {
    const wrapper = shallowMount(Component, {
      propsData: {
        value: true,
        label: "test"
      },
      localVue
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
