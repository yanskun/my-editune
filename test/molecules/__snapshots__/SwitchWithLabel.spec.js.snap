// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SwitchWithLabel snapshot 1`] = `
<div
  class="switch-with-label"
>
  <app-switch-stub
    class="switch"
  />
  <p
    class="label"
  >
    
  </p>
</div>
`;
